core/src/locktower.rs

```diff
@@ -162,8 +162,17 @@ impl Locktower {
                 };
                 Self::update_ancestor_lockouts(&mut stake_lockouts, &vote, ancestors);
             }
-            // each account hash a stake for all the forks in the active tree for this bank
-            Self::update_ancestor_stakes(&mut stake_lockouts, bank_slot, lamports, ancestors);
+
+            if vote_state.votes.len() > 1 {
+                let last_unsimulated_vote = &vote_state.votes[vote_state.votes.len() - 2];
```

Comment:
nit: last_real_vote or something not negative :)

Reply:
There is a VoteState method for this, nth_recent_vote